### ISSUE TITLE
Show the GeoCAT overview content on the homepage.

### ISSUE THREAD

**Issue by erogluorhan** [NCAR/geocat-tutorials#3]
The GeoCAT overview content is currently housed in `notebooks/geocat/00-Overview.ipynb`. We need to show that content on the tutorial home page. 

I can thnik of two ways:

1. Copy content from that notebook into `docs/index.rst`, remove the notebook
2. Incude that notebook in `docs/index.rst` if there is an effective way for that

Thoughts or alternative ideas? cc @anissa111 @jukent

**Comment by anissa111** [comment]
@erogluorhan Yes! I stumbled across a nice solution in `nbsphinx` [documentation](https://buildmedia.readthedocs.org/media/pdf/nbsphinx/latest/nbsphinx.pdf) in Section 12.

We can create toctrees in our notebooks instead of using a .rst "landing page"

**Comment by anissa111** [comment]
Follow up: turns out that was taking a lot of time to debug. I've implemented a solution similar to something we do in geocat-comp and uxarray docs where I've removed the landing pages and separated content with multiple toctrees with headers in the primary `index.rst`.

This allows the first document seen in each section to be the first notebook on the toctree, which is a simpler solution in the end anyway.
